src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -453,7 +457,11 @@ class ContentExplorer extends Component {
         return (
             // eslint-disable-next-line jsx-a11y/no-static-element-interactions, jsx-a11y/click-events-have-key-events
             <div
-                className={classNames('content-explorer', className)}
+                className={classNames(
+                    'content-explorer',
+                    isResponsive ? 'content-explorer--isResponsive' : '',
+                    className,
+                )}
```

Comment:
Same comment as the modal component.

```suggestion
                className={classNames('content-explorer', className, {
                    'bdl-ContentExplorer--responsive': isResponsive,
                })}
```

Reply:
Simplified and updated style to use SUIT naming.